modules/order/tests/src/Functional/OrderRefreshSettingsTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\Tests\commerce_order\Functional;
+
+use Drupal\Tests\commerce\Functional\CommerceBrowserTestBase;
+use Drupal\commerce_order\Entity\OrderType;
+
+/**
+ * Tests the cart refresh.
```

Comment:
Tests the order refresh settings.

(Why isn't this test a part of the OrderTypeTest?)